src/NHSD.BuyingCatalogue.Identity.UserDatabase/Tables/Organisations.sql

```diff
@@ -1,12 +1,12 @@
-﻿CREATE TABLE [dbo].[Organisations]
+﻿CREATE TABLE dbo.Organisations
 (
-	OrganisationId UNIQUEIDENTIFIER NOT NULL,
-    [Name] NVARCHAR(255) NOT NULL,    
-    [Address] NVARCHAR(MAX) NULL,
-    OdsCode NVARCHAR(8) NULL,
-    PrimaryRoleId NVARCHAR(8) NULL,
-    CatalogueAgreementSigned BIT NOT NULL,
-    LastUpdated DATETIME2(7) NOT NULL,
+	OrganisationId uniqueidentifier NOT NULL,
+    [Name] nvarchar(255) NOT NULL,
+    [Address] nvarchar(max) NULL,
+    OdsCode nvarchar(8) NULL,
+    PrimaryRoleId nvarchar(8) NULL,
+    CatalogueAgreementSigned bit DEFAULT 0 NOT NULL,
+    LastUpdated datetime2(7) DEFAULT GETUTCDATE() NOT NULL,
```

Comment:
Ruth & I agreed to not include the 'DEFAULT GETUTCDATE()'. I assume this would effect all your scripts if this was removed? Or the scripts would have to be changed to add a last updated date & time?

Reply:
Yes, that's correct. All the scripts would need to be changed to set `LastUpdated` to `GETUTCDATE()` and `CatalogueAgreementSigned` to `0` (or `1`), which seems somewhat unnecessary when we can set an acceptable default.

In these two cases we can easily set an acceptable default and it also means it will avoid any issues if we still have any environments where these two columns don't yet exist in the database (as a new `NOT NULL` column without a `DEFAULT` obviously isn't a good combo if any data already exists).

I'm not sure we really gain anything by omitting the defaults.

Reply:
Defaults are easier to deal with in terms of applying an upgrade via dacpac to existing tables. My vote is for keeping them.

Reply:
Sounds good 👍